api/client/build.go

```diff
@@ -173,7 +173,7 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 			includes = append(includes, ".dockerignore", *dockerfileName)
 		}
 
-		if err = utils.ValidateContextDirectory(root, excludes); err != nil {
+		if err := utils.ValidateContextDirectory(root, excludes); err != nil {
```

Comment:
Recently saw such check as `utils.ValidateContextDirectory(root, excludes) != nil` :) It is awful, but never thought that you can do this.